canvas_modules/common-canvas/__tests__/common-properties/controls/oneofselect-test.js

```diff
@@ -522,10 +522,29 @@ describe("oneofselect with custom value allowed works correctly", () => {
 		let dropdownWrapper = wrapper.find("div[data-id='properties-oneofselect-custom']");
 		const dropdownInput = dropdownWrapper.find("input");
 		dropdownInput.simulate("click");
+		// Enter 'e' in input to check [one, three] are filtered using shouldFilterItem
+		dropdownInput.simulate("change", { target: { value: "e" } });
 		dropdownWrapper = wrapper.find("div[data-id='properties-oneofselect-custom']");
```

Comment:
Hi @caritaou I have added this line because after adding `shouldFilterItem` dropdownList is having `0` items, is this fine ?